vector/src/main/res/layout/item_tchap_room_invitation.xml

```diff
@@ -45,11 +45,12 @@
         android:layout_width="0dp"
         android:layout_height="12dp"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/roomInvitationAvatarContainer"
+        app:layout_constraintTop_toBottomOf="@+id/roomInvitationAvatarContainer"
```

Comment:
please remove those "+id" changes which are not necessary (seems to be only useful at the widget declaration level), otherwise that will introduce many view references in the code